test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,189 @@
+## Workload Benchmark for KubeStellar
+
+To generate the sample workload for KubeStellar performance experiments for long running tests proceed as following:
+
+1. Clone the following clusterloader2 repo. It's the tool used to deploy the benchmark workload in a KubeStellar environment:
+
+   ```bash 
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+   The script `test/performance/common/setup-clusterloader2.sh` uses the environment variable set above. 
```

Comment:
The reader has not been told about a script yet, so the referent of "the script" is not clear.

Reply:
Agreed. Then, I removed that sentence and moved the instruction to set the env variable `CL2_DIR` to step 2-b instead.